From 950c136f14b983717963d5b5ae647ca3aebddd01 Mon Sep 17 00:00:00 2001
From: Brian Gianforcaro <b.gianfo@gmail.com>
Date: Mon, 27 Dec 2021 23:38:11 -0800
Subject: [PATCH 09/14] serenity: Add a empty O_NDELAY symbol so the dev
 stressor will compile

Serenity doesn't yet support O_NDELAY, so just make it compile by making
the definition 0, so the bitmask will be meaningless.
---
 stress-dev.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/stress-dev.c b/stress-dev.c
index 1fab38e..9ab1b31 100644
--- a/stress-dev.c
+++ b/stress-dev.c
@@ -24,6 +24,12 @@
  */
 #include "stress-ng.h"
 
+#ifdef __serenity__
+    /* Serenity doesn't yet support O_NDELAY, so just make it compile */
+    #define  O_NDELAY (0)
+#endif
+
+
 static const stress_help_t help[] = {
 	{ NULL,	"dev N",	"start N device entry thrashing stressors" },
 	{ NULL,	"dev-ops N",	"stop after N device thrashing bogo ops" },
-- 
2.32.0

